WordPressKit/WordPressKit/ReaderTopicServiceRemote.m

```diff
@@ -294,6 +308,50 @@ - (NSString *)endpointUrlForPath:(NSString *)endpoint
 
 #pragma mark - Private Methods
 
+
+/**
+ Generate an Site Info Post Subscription object
+
+ @param subscription A dictionary object for the site subscription
+ @return A nullable Site Info Post Subscription
+ */
+- (RemoteReaderSiteInfoSubscriptionPost *)postSubscriptionFor:(NSDictionary *)subscription
+{
+    if (![subscription wp_isValidObject]) {
```

Comment:
Question for you: I do appreciate the extra safety, but... are you adding this because you actually saw a case in which this is *needed*, or just for safety's sake?

Reply:
Because a subscription might <null> from the JSON response. So, rather than have `subscription == nil || [subscription isEqual:[NSNull null]]`, I imported that category to check both the cases.

Reply:
Thank you for the feedback sir!!

Reply:
I'm fine with the safety check, but I don't think `wp_isValidObject` is the right one. We don't just care that it's not a `NSNull`, but we'll want to ensure it's a `NSDictionary` or I think  the following call to `dictionaryForKey` will crash, right? (all the Obj-C knowledge is staring to fade away 😢)